Documentation/nfs.md

```diff
@@ -279,6 +346,64 @@ After the NFS server pod is running, follow the same instructions from the previ
 
 After that, follow the rest of the instructions in the [Accessing the Export](nfs.md#accessing-the-export) section and then the [Consuming the Export](nfs.md#consuming-the-export) section to consume the NFS volume.
 
+## Dynamic provisioning with NFS
```

Comment:
I'm thinking that these dynamic provisioning steps should replace the existing flow that uses static provisioning that is earlier in the document.  Lets make dynamic provisioning the primary recommended approach.

The sections i'm talking about replacing are titled "Accessing the Export" and "Consuming the Export" and have language like "To access the export from another pod, you must first manually create a PersistentVolume with the connection information. This experience is a bit cumbersome, but will be improved in the future with dynamic provisioning support.".

Let's remove those old sections and just make dynamic provisioning the only flow we talk about.

Reply:
yeah makes sense

Reply:
done